script.js

```diff
@@ -4,26 +4,35 @@
  * @link https://developer.mozilla.org/en-US/docs/Web/API/Element/querySelectorAll
  */
 
-import Backpack from "./Backpack.js";
-
-const everydayPack = new Backpack(
+import Closet from "./Closet";
+
+// Wall_Dimentions, Items, boxes, Wall_color, Door,
+const My_Closet = new Closet(
+  "4ft Length, 8ft Height, 2ft Deep",
+  "Clothing, boxes, safe, shelf",
+  "Clothing and shoes",
+  "light blue",
+  true
 );
 
 const content = `
-     <figure class="backpack__image">
-       <img src= alt="" />
+     <figure class="Closet__image">
+       <img src= ${My_Closet.image}alt="" />
```

Comment:
this is the same issue oscar had. it should be <img src="${My_Closet.image} alt="" />. The space between src= and the template literal will cause it to break.